plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialLevel.java

```diff
@@ -50,41 +51,67 @@ public static void registerTags() {
         // @returns ElementTag(Number)
         // @group properties
         // @description
-        // Returns the maximum level for a levelable material (like water, lava, and Cauldrons), a cake, or a beehive.
+        // Returns the maximum level for a valid material.
+        // Valid materials include water, lava, cauldrons, cake, beehives, and snow.

```

Comment:
Please change this back to the way it was. `a levelable material (like water, lava, and Cauldrons)` is NOT a definitive / complete list.